pkg/errors/machines.go

```diff
@@ -53,6 +53,13 @@ func CreateMachine(msg string, args ...interface{}) *MachineError {
 	}
 }
 
+func UpdateMachine(msg string, args ...interface{}) *MachineError {
```

Comment:
Not sure I understand this function, and it is possibly the name that makes it confusing. Calling this function always returns an error. It does no updating. What am i missing?

Reply:
It's under `pkg/errors` so the purpose of the function is to always return update machine error.

Reply:
I see. I was going to suggest `errors.UpdateMachineError()` but I don't know if that stutters too much. However, adding `Error` to the name indicates that it is generate a `MachineError`. Thoughts?

Reply:
If we are just following the existing pattern then so be it...